experiments/run_lbann_experiment.sh

```diff
@@ -371,6 +371,7 @@ case ${CLUSTER} in
         ;;
 esac
 echo "export MV2_USE_RDMA_CM=0"                         >> ${BATCH_SCRIPT}
```

Comment:
You may want to add the `export MV2_USE_RDMA_CM=0` to the other three scripts, at least for now.